packages/engine-server/src/markdown/remark/blockAnchors.ts

```diff
@@ -65,28 +68,37 @@ function attachParser(proc: Unified.Processor) {
   inlineMethods.splice(inlineMethods.indexOf("link"), 0, "blockAnchor");
 }
 
-function attachCompiler(proc: Unified.Processor, _opts?: CompilerOpts) {
+function attachCompiler(proc: Unified.Processor, _opts?: PluginOpts) {
   const Compiler = proc.Compiler;
   const visitors = Compiler.prototype.visitors;
 
   if (visitors) {
-    visitors.blockAnchor = function (node: BlockAnchor) {
+    visitors.blockAnchor = function (node: BlockAnchor): string | Element {
+      debugger;
       const { dest } = MDUtilsV4.getDendronData(proc);
       switch (dest) {
         case DendronASTDest.MD_DENDRON:
         case DendronASTDest.MD_REGULAR:
         case DendronASTDest.MD_ENHANCED_PREVIEW:
-        case DendronASTDest.HTML: {
-          // Anything more to do here? Can we embed HTML into the preview?
-          // Prints ^{\^block-id} so that it gets rendered small.
-          return `^\{\\^${node.id}\}`;
-        }
+        case DendronASTDest.HTML:
```

Comment:
this is legacy from days past when we used to use `remark` to render HTML`. this should be a dead end at this point since rendering HTML means using rehype